code/_helpers/names.dm

```diff
@@ -86,63 +86,63 @@ var/syndicate_code_response//Code response for traitors.
 	-N
 	*/
 
-/proc/generate_code_phrase()//Proc is used for phrase and response in master_controller.dm
+// Helper. Called in misc_late.dm for late misc init.
+/proc/populate_code_phrases(override = FALSE)
+	if (override || !syndicate_code_phrase)
+		syndicate_code_phrase = generate_code_phrase()
+	if (override || !syndicate_code_response)
+		syndicate_code_response = generate_code_phrase()
 
-	var/code_phrase = ""//What is returned when the proc finishes.
-	var/words = pick(//How many words there will be. Minimum of two. 2, 4 and 5 have a lesser chance of being selected. 3 is the most likely.
+/proc/generate_code_phrase() //Proc is used for phrase and response in master_controller.dm
+
+	var/code_phrase = "" //What is returned when the proc finishes.
+	var/words = pick( //How many words there will be. Minimum of two. 2, 4 and 5 have a lesser chance of being selected. 3 is the most likely.
 		50; 2,
 		200; 3,
 		50; 4,
 		25; 5
 	)
 
-	var/safety[] = list(1,2,3)//Tells the proc which options to remove later on.
+	var/safety[] = list(1,2,3) //Tells the proc which options to remove later on.
 	var/nouns[] = list("love","hate","anger","peace","pride","sympathy","bravery","loyalty","honesty","integrity","compassion","charity","success","courage","deceit","skill","beauty","brilliance","pain","misery","beliefs","dreams","justice","truth","faith","liberty","knowledge","thought","information","culture","trust","dedication","progress","education","hospitality","leisure","trouble","friendships", "relaxation")
 	var/drinks[] = list("vodka and tonic","gin fizz","bahama mama","manhattan","black Russian","whiskey soda","long island tea","margarita","Irish coffee"," manly dwarf","Irish cream","doctor's delight","Beepksy Smash","tequilla sunrise","brave bull","gargle blaster","bloody mary","whiskey cola","white Russian","vodka martini","martini","Cuba libre","kahlua","vodka","wine","moonshine")
-	var/locations[] = teleportlocs.len ? teleportlocs : drinks//if null, defaults to drinks instead.
-
-	var/names[] = list()
-	for(var/datum/data/record/t in data_core.general)//Picks from crew manifest.

```

Comment:
This shit never triggered so I nuked it. I'm unsure if it's a good idea or not, but there's no real reason for code phrases to be generated later on in the round.